src/components/Common/common.css

```diff
@@ -11,6 +11,32 @@ select {
     @apply lil-rounded-lg lil-p-3;
 }
 
+/*
+* - border-spacing not supported in tailwind
+*/
+.lil-border-spacing-0 {
+    border-spacing: 0;
+}
+
+/*
+* to make rounded corners by assigning border-radius for each corner cell
+*/
+.lil-thead tr:first-child th:first-child {
+    @apply lil-rounded-tl-xl;
+}
+
+.lil-thead tr:first-child th:last-child {
+    @apply lil-rounded-tr-xl;
+}
+
+.lil-tbody tr:last-child td:first-child {
+    @apply lil-rounded-bl-xl;
+}
+
+.lil-tbody tr:last-child td:last-child {
+    @apply lil-rounded-br-xl;
+}
+
```

Comment:
i think this is not needed...consumer can assign border
also if it can't be avoided it should be outside common with styling for table instead of thead, tbody